hardhat/scripts/create-proposals.ts

```diff
@@ -45,7 +46,9 @@ async function main() {
     const formData = createNewFormData(type);
     const votingApp = new ethers.Contract(api3Voting[formData.type], votingAbi, account);
     try {
-      await votingApp.newVote(encodeEvmScript(formData, api3Agent), encodeMetadata(formData), true, true);
+      const goEncodeEvmScript = encodeEvmScript(formData, api3Agent);
+      if (!isGoSuccess(goEncodeEvmScript)) throw new Error();
```

Comment:
Worth including some kind of message with this error?

```suggestion
      if (!isGoSuccess(goEncodeEvmScript)) throw new Error('Unable to encode form data');
```